tasks/install.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Add Apt Key for NodeJS PPA
+  apt_key:
+    id: 1655A0AB68576280
+    keyserver: keyring.debian.org
+
+- name: Add NodeJS 14 PPA
+  apt_repository:
+    repo: "deb https://deb.nodesource.com/node_14.x {{ ansible_distribution_release }} main"
+    filename: nodesource
+
+- name: Install npm
+  apt:
+    name:
+      - nodejs
+
+- name: Add build user
+  user:
+    name: build-user
+
+- name: Download archive
+  get_url:
+    url: "https://github.com/Johni0702/mumble-web/archive/{{ version }}.tar.gz"
+    dest: "{{ archive_file }}"
+    checksum: "{{ checksum }}"
+  become: False
+  delegate_to: localhost
+
+- name: Clean and create build directory
+  file:
+    path: "{{ build_dir }}"
+    state: "{{ item }}"
+  with_items:
+    - absent
+    - directory
+
+- name: Unpack archive
+  unarchive:
+    src: "{{ archive_file }}"
+    dest: "{{ build_base }}"
+    owner: build-user
+
+- name: Build mumble-web
+  become_user: build-user
+  command:
+    cmd: npm install
+    chdir: "{{ build_dir }}"
+
+- name: Create deployment parent directory
+  file:
+    path: "{{ deploy_base }}"
+    state: directory
```

Comment:
No owner and permissions?

Reply:
Default seems to be `755` and `root:root` which is typical for this directory. 
I will add it here to make it explicit.

Reply:
Actually I just noticed this is already done in the [main task](https://github.com/stuvusIT/mumble-web/pull/1/files#diff-2444ad0870f91f17ca6c2a5e96b26823R17-R20)